src/client/features/view/components/menu/search.js

```diff
@@ -0,0 +1,96 @@
+const _ = require('lodash');
+
+//Apply hover styling to a collection of nodes
+function updateStyling(style, matched, cy) {
+  _.forEach(matched, node => {
```

Comment:
Is the ```matched``` parameter a cy.js collection?

If so, you should rename ```matched``` to ```matchedNodes```
and then use ```nodes.forEach``` instead of ```lodash.forEach```

i.e

```js
  matchedNodes.forEach(node => {
```

Reply:
Yes, it is a collection. I have renamed it and used regular forEach().